js/data.js

```diff
@@ -49,35 +49,46 @@ const _FACTIONS = {
 
 const _MAPS = {
   WOODLAND: {
-    name: "Woodland",
+    name: "Fall",
+    alt: "The Fall Root map.",
+    fileName: "fall.png",
     numClearings: 12
   },
   WINTER: {
     name: "Winter",
+    alt: "The Winter Root map.",
+    fileName: "winter.png",
     numClearings: 12
   },
   LAKE: {
     name: "Lake",
+    alt: "The Lake Root map.",
+    fileName: "lake.png",
     numClearings: 12
   },
   MOUNTAIN: {
     name: "Mountain",
+    alt: "The Mountain Root map.",
+    fileName: "mountain.png",
     numClearings: 12
   }
 }
 
 const _CLEARING_TYPES = {
   FOX: {
     name: "Fox",
-    color: "#FF0000"
+    color: "#FF0000",
```

Comment:
Now that we have clearing-specific CSS, we should probably actually pull out "color" into a CSS class, to try to maintain some semblance of a divide between data and view.

Reply:
That's a good idea.